### PR TITLE
docs(README): Update links

### PR DESCRIPTION
**Pull request by nnobelis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -287,7 +287,7 @@ justify the resolution on a global scale.
 |-------------|--------|-----------------------------------|
 | YAML / JSON | Global | `$ORT_CONFIG_DIR/resolutions.yml` |
 
-#### [Repository configuration file](./docs/config-file-ort-yml.md)
+#### [Repository configuration file](./website/docs/configuration/ort-yml.md)
```
Comment (sschuberth) [inline README.md:290]: Looks like all title links in this section starting with "Copyright garbage file" are broken. Mind fixing them as well?

**Comment by nnobelis** [comment]
@sschuberth  The tests failures are related to link I didn't change. How do you understand that ?

**Comment by sschuberth** [comment]
> @sschuberth The tests failures are related to links I didn't change. How do you understand that ?

I believe the *markdown-links* failures are valid, though probably not related to your changes. Mind still fixing these, too?

The *funTest-docker* failure is unrelated, but should disappear now on a rebase.

File: README.md (hunk 1 above)
Comment (fviernau) [reply README.md:290]: I've fixed some https://github.com/oss-review-toolkit/ort/pull/7700.

Comment (sschuberth) [reply README.md:290]: Actually @nnobelis [had already fixed the other links](https://github.com/oss-review-toolkit/ort/compare/64da439b0191e50a44caee39a873a844bc70abcb..0061f6b0f730ccc0026027cc2efaa6cb4ea89482) in this PR. Let's take this PR from him and rebase yours on top.

**Pull request merged**